README.md

```diff
@@ -53,7 +53,8 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 ```json
 {
 	"port": 8080,
-	"bugtracking": true
+	"bugtracking": true,
+	"diffType": "textdiff" | "sidebysidediff"
```

Comment:
An example in readme should be valid and this is a simple small example of the rc file.  Can we please revert this change?